includes/modules/payment/paypal/PayPalRestful/Webhooks/WebhookController.php

```diff
@@ -20,6 +20,8 @@ class WebhookController
 {
     public function __invoke(): bool|null
     {
+        defined('TABLE_PAYPAL_WEBHOOKS') or define('TABLE_PAYPAL_WEBHOOKS', DB_PREFIX . 'paypal_webhooks');
```

Comment:
Maybe use `zen_define_default`?

Reply:
I'd considered that, but I'm also somewhat aiming for older ZC versions that don't have that function.